packages/ui-components/src/components/portal/portal.tsx

```diff
@@ -97,8 +97,10 @@ export class KvPortal implements IPortal, IPortalEvents {
 		computePosition(this.reference, this.portal, this.getOptions()).then(({ x, y, placement, middlewareData }) => {
 			if (this.autoUpdate) {
 				const { referenceHidden } = middlewareData.hide;
-				this.visible = !referenceHidden;
-				this.portal.style.zIndex = referenceHidden ? `${PORTAL_Z_INDEX.hidden}` : `${this.zIndex}`;
+				if (this.show) {
```

Comment:
This here, is it to prevent that dropdown from closing?

Reply:
Actually is the opposite 😅
This component had an issue when we were controlling the open state from outside, sometimes it opened because the `computePosition` was being called since these portals are never removed from the DOM. This is to prevent that behaviour.